docs/API_reference.md

```diff
@@ -580,6 +581,24 @@ Return a POST request JSON response, if one of the requests fails return the err
 If the **value_id** field is not filled the method will return: `{'error': 'missing_parameter'}`.
 In case the language within the `lang` field of the JSON is invalid the method will return `{'error': 'invalid_language'}`.
 
+#### plenty_api_create_property_selection_name <a name=create-property-selection></a>
+
+Create a new name for a property selection value and input the initial value for that language.
```

Comment:
I think you can reduce the description above to:
`Create a new name for an existing property selection value for the given language`